src/main/java/backend/UpdateController.java

```diff
@@ -144,4 +146,40 @@ private void openRepositoriesInFilters(List<FilterExpression> filterExprs) {
         // No sort or updated, return sort by descending ID, which is the default.
         return Qualifier.getSortComparator(models, "id", true, false);
     }
+
+    /**
+     * Applies the count qualifier to the set of issues obtained after the sorting order is applied.
+     *
+     * @param issueList Issue list obtained after filtering and sorting.
+     * @param expression The filter expression of the particular panel.
+     * @return The modified issue list with the count applied.
+     */
+    private List<TurboIssue> applyCountQualifierToIssueList(List<TurboIssue> issueList, FilterExpression expression){
```

Comment:
These two functions can be combined into one that just returns the number of elements to `limit` the stream to.

Also, instead of throwing an exception, maybe just use the first one, like `sort` does?